[EDU-3699] refactor: update guides marketplace - EN (#638)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/integrations/request-variation/index.mdx

With the help of this integration, a signed cookie can be created or updated, containing details about how many different requests were made in succession to an edge application.

1. Access [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Marketplace**.
2. On the Marketplace homepage, select the search box and type “*variation*” or browse through the cards to find the two **Request Variation Controller** integrations.
3. Once you've found the **Request Variation Controller - Hash Generator** and **Request Variation Controller - Hash Validator** cards, select one to go to the integration's page.
4. On the integration page, look for the **Subscribe for** section on the bottom-right corner.
5. Click the **Get It Now** button.

You'll follow the same steps to install the second integration.

In both cases, after clicking on the **Get it now** button, you'll see a message indicating that your integration was successfully installed.

---

## Configuring the integration

Once you've gotten your integration on **Marketplace**, it'll be available at your edge functions list. To use the integration, you'll have to create a new edge application.

:::tip

> Go to the [building](/en/documentation/products/guides/build/build-an-application/) section for more details on how to build an edge application.

Reviewer: tá diferente do outro que sugeri tirar o more details

> Go to the [building](/en/documentation/products/guides/build/build-an-application/) section for more details on how to build an edge application.

Author: ```suggestion
To create an edge application, go to the [building](/en/documentation/products/guides/build/build-an-application/) section.
```

3. Choose an easy to remember name for your function.
4. On the dropdown function menu, select the *Variation Request* function.

> This action will load the function, showing a form with the function code and, just above it, two tabs: **Code** and **Args**. By clicking on the **Code** tab you will be able to navigate through the code, but won't be able to change it.

Reviewer: ```suggestion
This action will load the function, showing a form with the function code and, just above it, two tabs: **Code** and **Args**. By clicking on the **Code** tab, you'll be able to navigate through the code, but won't be able to change it.
```
